src/main/java/Wolox/training/models/Book.java

```diff
@@ -35,6 +36,9 @@
     @Column(nullable = false)
     private String isbn;
 
+    @ManyToMany(mappedBy = "books")
+    private List<User> users;
```

Comment:
It is good practice to initialize collections, to avoid repeating null checking elsewhere in the code
```suggestion
    private List<User> users = new ArrayList<>();
```